components/tabs/src/tabs/tabs.styles.ts

```diff
@@ -4,6 +4,7 @@ import { makeStyles, shorthands } from '@griffel/react';
 export const useTabsStyles = makeStyles({
   root: {
     color: tokens.colorNeutralForeground1,
+    // @ts-ignore Griffel is not supporting 1 as a value for CSS variables.
```

Comment:
Does it just want strings

Reply:
@brandonthomas, I need to file this on the Griffel team. That tying is incorrect.